src/Test.elm

```diff
@@ -0,0 +1,212 @@
+module Test exposing (..)
```

Comment:
Don't name your sources `Test` - tests are automated checks we'll get to in the near future. This kind of file can be aptly named `Playground` 🙂